lamindb/__init__.py

```diff
@@ -85,7 +85,7 @@
 
 """
 
-__version__ = "0.50.0"  # denote a release candidate for 0.1.0 with 0.1rc1
+__version__ = "0.50.1"  # denote a release candidate for 0.1.0 with 0.1rc1
```

Comment:
@Koncopd - this can only be bumped to a release when we make a release. The bump here happens at the same time as the PyPI upload if all integration tests pass.

If you'd like to change this number during the dev cycle, you can only change it to a pre-release like 0.50.1a1 or something similar.

But we haven't indicated pre-releases on staging in the past months - would you prefer to do it going forward?

Reply:
sorry, will do like you say.

Reply:
No worries!